src/main/scala/no/ndla/articleapi/model/domain/Article.scala

```diff
@@ -16,8 +16,8 @@ import org.json4s.FieldSerializer._
 import org.json4s.native.Serialization._
 import scalikejdbc._
 
-case class Article(id: Option[Long],
-                   revision: Option[Int],
+case class Article(_id: Option[Long],
+                   _revision: Option[Int],
```

Comment:
For å kunne aksessere id og revision uten å måtte skrive `id.get` over alt, var tanken i første omgang å override getter funksjonen som case classen automatisk genererer, til å returnere selve verdien (slik de er definert i `id` og `revision` funksjonene i denne klassen). Problemet er at det ikke er mulig å override dem, derfor ble id og revision prefixet med en underscore, slik det er foreslått her https://www.safaribooksonline.com/library/view/scala-cookbook/9781449340292/ch04s07.html